walk-in-interview/src/main/webapp/business-jobs-list/script.js

```diff
@@ -0,0 +1,172 @@
+const JOB_DETAILS_PATH = '../job-details/index.html';
+const HOMEPAGE_PATH = '../index.html';
+
+// TODO(issue/34): implement pagination for job listings
+
+window.onload = () => {
+  loadAndDisplayJobListings();
+}
+
+/**
+ * Add the list of jobs made.
+ */
+async function loadAndDisplayJobListings() {
+  const backButton = document.getElementById('back');
+  backButton.innerText = STRINGS['back'];
+  backButton.addEventListener('click', (_) => {
+    window.location.href = HOMEPAGE_PATH;
+  });
+
+  const jobListingsTitle =
+    document.getElementById('job-listings-title');
+  jobListingsTitle.innerText = STRINGS['job-listings-title'];
+
+  const jobPageData = 
+    await getJobsMade(DEFAULT_PAGE_SIZE, DEFAULT_PAGE_INDEX)
+      .catch((error) => {
+        console.error('error fetching job listings', error);
+        setErrorMessage(/* errorMessageElementId= */ 'error-message',
+            /* msg= */ STRINGS['get-jobs-error-message'],
+            /* include default msg= */ false);
+      });
+
+  displayJobListings(jobPageData);
+}
+
+/**
+ * Makes GET request to retrieve all the job posts made
+ * by the current business user. 
+ * This function is called when the interest page is loaded.
+ * 
+ * @param {int} pageSize The number of jobs for one page.
+ * @param {int} pageIndex The page index (starting from 0).
+ * @return {Object} The data returned from the servlet.
+ */
+function getJobsMade(pageSize, pageIndex) {
+  const params = `pageSize=${pageSize}&pageIndex=${pageIndex}`;
+
+  fetch(`${API['business-jobs-list']}?${params}`, {
+    method: 'GET',
+    headers: {'Content-Type': 'application/json'},
+  })
+      .then((response) => response.json())
+      .then((data) => {
+        console.log('data', data);
+        // Resets the error (there might have been an error msg from earlier)
+        setErrorMessage(/* errorMessageElementId= */ 'error-message',
+            /* msg= */ '', /* includesDefaultMsg= */ false);
+        return data;
+      });
+}
+
+/**
+ * This will add all the job listings onto the page.
+ * 
+ * @param {Object} jobPageData The details to be shown on the page.
+ */
+function displayJobListings(jobPageData) {
+  const jobListingsElement = document.getElementById('job-listings');
+  const jobShowing = document.getElementById('job-listings-showing');
+
+  /* reset the list so we don't render the same jobs twice */
+  jobListingsElement.innerHTML = '';
+  jobShowing.innerHTML = '';
+
+  if (jobPageData === undefined ||
+    !jobPageData.hasOwnProperty('jobList') ||
+    jobPageData['jobList'].length === 0) {
+    setErrorMessage(/* errorMessageElementId= */ 'error-message',
+        /* msg= */ STRINGS['no-jobs-error-message'],
+        /* includesDefaultMsg= */ false);
+    return;
+  }
+
+  const jobListings = jobPageData['jobList'];
+
+  jobListings.forEach((job) => {
+    jobListingsElement.appendChild(buildJobElement(job));
+  });
+
+  jobShowing.innerText = `${jobPageData['range'].minimum} -` +
```

Comment:
same comment as PR#85, you cannot construct strings by code as it won't work in a localized environment (meaning multiple languages can be used). please double check jobShowing, jobAddress, jobPay, requirements list